app/tests/mock_database.py

```diff
@@ -0,0 +1,141 @@
+import uuid
+
+from app.main.db.models import Body, Consignment, File, FileMetadata, Series, db
+
+
+def generate_mock_data():
+    # first record
```

Comment:
Also, not for this PR, but we should definitely start using something like https://factoryboy.readthedocs.io/en/stable/orms.html#sqlalchemy to create test db objects. (I have used it before for Django and its great)

This will allow us to abstract away object creation to the factories and reduce the amount of code we need to write due to the nested nature of the factory creation and default values.

Reply:
created https://national-archives.atlassian.net/browse/AYR-512 to add it in if not done otherwise